src/main/java/frc/robot/subsystem/Elevator.java

```diff
@@ -0,0 +1,101 @@
+package frc.robot.subsystem;
+
+import frc.robot.Constants;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import com.revrobotics.RelativeEncoder;
+
+import edu.wpi.first.math.controller.ProfiledPIDController;
+
+enum ElevatorStates {
+    HIGH(Constants.Elevator.HIGH_HEIGHT, !Constants.Elevator.HIGH_GEAR), 
+    MIDDLE(Constants.Elevator.MIDDLE_HEIGHT, Constants.Elevator.HIGH_GEAR), 
+    LOW(Constants.Elevator.LOW_HEIGHT, Constants.Elevator.HIGH_GEAR); 
+
+    private double targetHeight; 
+    private boolean highGear; 
+
+    ElevatorStates(double targetHeight, boolean highGear) {
+        this.targetHeight = targetHeight; 
+        this.highGear = highGear; 
+    }
+
+    public double getTargetHeight() {
+        return targetHeight; 
+    }
+
+    public boolean getHighGear() {
+        return highGear; 
+    }
+}
+
+public class Elevator {
+    private CANSparkMax leftMotor; 
```

Comment:
aren't there 4 motors all beign 775s? (would be controlled by victors or talonfxs (if we said neos then keep using that, you might need 4 for the different gears tho). Ask in main

Reply:
recommended to use two (neos or krakens) since we won't use diff gears